### PR TITLE
Doesn't support keys with a leading forward slash `/`' for GET

### LINKED ISSUE #21: Doesn't support keys with a leading forward slash `/`' for GET
To reproduce:

1. Add a key in the explorer with a trailing forward slash `/` - e.g., `/testing`. This operation succeeds
2. Trey and retrieve the value if the key, This will produce an error in the log `ERROR: cannot get key: no such key`, and no value is shown in the explorer

Tested with `0.9.0`

### PR DESCRIPTION
**Pull request by mikkelhegn**
Fixes #21

You can now use any character for keys and values, and they will be rendered correctly in the explorer.

Also upgraded to latest SDK for Go

### CODE REVIEW

File: explorer/main.go [hunk 1]
```diff
@@ -69,16 +73,16 @@ func getBasePath(h http.Header) string {
 }
 
 // Setup the router and handle the incoming request.
-func serve(w http.ResponseWriter, r *http.Request) {
-	user, pass, err := GetCredentials()
+func serve(w http.ResponseWriter, r *http.Request, logger *log.Logger) {
```
Comment (rajatjindal) [inline explorer/main.go]: do we need to pass logger like this here? I would think the logger we declared above should work

File: explorer/main.go [hunk 2]
```diff
@@ -95,29 +99,29 @@ func serve(w http.ResponseWriter, r *http.Request) {
...
+func UIHandler(w http.ResponseWriter, _ *http.Request, _ spinhttp.Params) {
 	out := strings.ReplaceAll(HTMLTemplate, "{{.SpinRoute}}", spinRoute)
 	w.Write([]byte(out))
 
 }
 
 // ListKeysHandler is the HTTP handler for a list keys request.
-func ListKeysHandler(w http.ResponseWriter, _ *http.Request, p spin.Params) {
+func ListKeysHandler(w http.ResponseWriter, _ *http.Request, p spinhttp.Params) {
+	logger = log.New(os.Stderr, "", log.LstdFlags)
```
Comment (rajatjindal) [inline explorer/main.go]: I'll check once, but re-initializing logger may not be required.

Comment (rajatjindal) [reply explorer/main.go]: yeah, i checked. we don't need to initialize the logger again here.

File: explorer/main.go [hunk 3]
```diff
@@ -127,23 +130,23 @@ func ListKeysHandler(w http.ResponseWriter, _ *http.Request, p spin.Params) {
...
-	defer kv.Close(store)
 
 	start := time.Now()
-	value, err := kv.Get(store, key)
-	log.Printf("GET operation took %s", time.Since(start))
+	value, err := store.Get(string(safeKey))
+	logger.Printf("GET operation took %s", time.Since(start))
 	if err != nil {
-		log.Printf("ERROR: cannot get key: %v", err)
+		logger.Printf("ERROR: cannot get key: ", err)
```
Comment (rajatjindal) [inline explorer/main.go]: %v is missing here

**Review by radu-matei — APPROVED** [review]

**Review by adamreese — APPROVED** [review]

**Pull request merged**